hbase-protocol-shaded/src/main/protobuf/server/region/WAL.proto

```diff
@@ -32,6 +32,8 @@ message WALHeader {
   optional bool has_tag_compression = 3;
   optional string writer_cls_name = 4;
   optional string cell_codec_cls_name = 5;
+  optional bool has_value_compression = 6;
+  optional uint32 value_compression_codec = 7;
```

Comment:
nit: call this attribute `value_compression_type` so that it matches the field in `AbstractProtobufLogWriter`, or rename that field to `valueCompressionCodec` so that it matches this attribute. Or call it `value_compression_algorithm` and update the property name down in `ReaderBase`, so that everything aligns with the name of the Java type, which is a `o.a.h.h.io.compress.Compression$Algorithm`.

Reply:
`value_compression_algorithm` it is